addon/components/cfb-form-editor/question.js

```diff
@@ -189,6 +189,13 @@ export default Component.extend(ComponentQueryManager, {
     return namespace ? `${namespace}-` : "";
   }),
 
+  requiredIsVisible: computed("changeset.{__typename,isRequired}", function() {
+    return (
+      (this.model.isRequired === "true" || this.model.isRequired === "false") &&
```

Comment:
Shouldn't this be `this.changeset`?

Reply:
Indeed. And it solves the problem.

The `this.model` was from my first attempt and I didn't test but went directly to Google. Thank you!